mobileapp/www/js/index.js

```diff
@@ -222,21 +234,37 @@ var app = {
       app.login();
     });
 
+    $("#use-touchid").click(function (e) {
+      if (window.localStorage.touchidLogin == 0) {
+        window.localStorage.setItem('touchidLogin', 1);
+        $('#use-touchid').html("Turn off TouchID");
+        app.alert("TouchID Activated");
+      } else {
+        window.localStorage.setItem('touchidLogin', 0);
+        $('#use-touchid').html("Turn on TouchID");
+        app.alert("TouchID De-Activated");
+      }
+    });
+
     $('#forget-credentials').click(function (e) {
-      app.keyChain.removePassphrase(function (err) {
-	if (err) {
-	  console.error(err);
-	  app.alert('There is no passphrase to remove from keychain', 'warning');
-	} else {
-	  app.alert('Passphrase removed!', 'info');
-	}
-	delete window.localStorage.lastUserLogin;
-	// re-set the login screen
-	$('#username-login').show();
-	$('#username-placeholder').html('').hide();
-	$('#password-login').show();
-	e.disabled = true;
-      });
+        if (window.localStorage.touchidLogin == 1) {
+          app.alert("Please disable TouchID before Forgetting Credentials");
```

Comment:
Right. It defaults to 'warning' if the second argument is not present. Which warning was my intended level.